plugins/source/digitalocean/docs/tables/digitalocean_domain_records.md

```diff
@@ -12,7 +12,7 @@ The primary key for this table is **id**.
 |_cq_sync_time|Timestamp|
 |_cq_id|UUID|
 |_cq_parent_id|UUID|
-|id (PK)|String|
+|id (PK)|Int|
```

Comment:
This is a breaking change in the DigitalOcean plugin correct?

Reply:
Yeah. though I dont think we have any usage for that plugin anyway per analytics.